randovania/games/dread/logic_database/Cataris.txt

```diff
@@ -3874,6 +3874,9 @@ Extra - asset_id: collision_camera_053
                   Heat/Cold Runs (Advanced) and Heat Damage ≥ 200
   > Dock to Underlava Puzzle Room 1
       Morph Ball and After Cataris - Underlava Speed Blocks Destroyed
+  > Event - Speed Blocks Destroyed
+      # Charge shinespark on above lava platform
+      Gravity Suit and Morph Ball and Speed Booster and Speed Booster Upgrade and After Cataris - Lava Grapple Block and Speed Booster Conservation (Beginner)
```

Comment:
Can you break the speed blocks at the dock to Underlava Puzzle Room 1?

Reply:
i would also test being able to get to the item with just speed without breaking those blocks to the left.

Reply:
ok, i tested, and yes you can destroy the speed blocks. and yes you can get up without destroying them (though irrelevant now). Since you need beginner conservation to get to that dock from reverse in the first place, it should be fine leaving the connection in Underlava Puzzle Room 1 as trivial.

Reply:
Yep